robottelo/decorators/func_shared/redis_storage.py

```diff
@@ -0,0 +1,72 @@
+# -*- encoding: utf-8 -*-
+try:
+    import redis
+except ImportError:
+    redis = None
+
+from robottelo.config import settings
+from robottelo.decorators.func_shared.base import (
+    BaseStorageHandler,
+    DEFAULT_LOCK_TIMEOUT,
+)
+
+REDIS_DEFAULT_HOST = 'localhost'
+REDIS_DEFAULT_HOST_PORT = 6379
+REDIS_DEFAULT_DB = 0
+REDIS_DEFAULT_PASSWORD = None
+
+if settings.configured:
+    REDIS_DEFAULT_HOST = settings.shared_function.redis_host
+    REDIS_DEFAULT_HOST_PORT = settings.shared_function.redis_port
+    REDIS_DEFAULT_DB = settings.shared_function.redis_db
+    REDIS_DEFAULT_PASSWORD = settings.shared_function.redis_password
```

Comment:
Be careful. Unfortunately calling configure on setting on import time cause some errors. @elyezer have pointed it out on some PRs